localization/xliff/enu/constants/localizedConstants.enu.xlf

```diff
@@ -284,6 +284,9 @@
         <trans-unit id="disconnectConfirmationMsg">
             <source xml:lang="en">Are you sure you want to disconnect?</source>
         </trans-unit>
+        <trans-unit id="noActiveEditorMsg">
+            <source xml:lang="en">A SQL query editor window must have focus before executing this command</source>
```

Comment:
>query editor window [](start = 40, length = 19)

you probably could shorten "SQL query editor window" to "SQL editor" without losing much in this error message